### ISSUE TITLE
inconsistent code on .<clinit>():void

### ISSUE THREAD

**Issue by jmue** [skylot/jadx#73]
the following code fails to decompile:

``` java
public class NativeClassLoader {

    private static boolean libraryPresent = false;
    private static Throwable libraryFailedException = null;

    static {
        try {
            System.loadLibrary("nativeLib");
            libraryPresent = true;
        } catch (SecurityException | UnsatisfiedLinkError var1) {
            libraryPresent = false;
            libraryFailedException = var1;
        }

    }
}
```

**Comment by jmue** [comment]
I think this one can be closed as of d55969b.

**Comment by asashour** [comment]
This can be closed.

The output is:

```
class TestClass {
    private static Throwable libraryFailedException;
    private static boolean libraryPresent;

    TestClass() {
    }

    static {
        libraryPresent = false;
        libraryFailedException = null;
        try {
            System.loadLibrary("nativeLib");
            libraryPresent = true;
        } catch (SecurityException | UnsatisfiedLinkError var1) {
            libraryPresent = false;
            libraryFailedException = var1;
        }
    }
}

```
